partition_table_engine/src/partition.rs

```diff
@@ -169,37 +164,33 @@ impl Table for PartitionTableImpl {
                 table: self.get_sub_table_ident(partition),
```

Comment:
It seems `self.get_sub_table_ident` will be called L159 and L164. How about calling it just once?